VideoWeb/VideoWeb/ClientApp/src/app/app.component.html

```diff
@@ -15,6 +15,7 @@
       [ngClass]="(backLinkDetails$ | async)?.className"
     >
       <app-beta-banner *ngIf="loggedIn" [isRepresentativeOrIndividual]="isRepresentativeOrIndividual"></app-beta-banner>
+      <app-cookie-banner></app-cookie-banner>
```

Comment:
Take a look at the GDS website for an example of placement
<img width="2873" alt="image" src="https://github.com/user-attachments/assets/66906e24-1d3c-462c-95d3-65c7740e7e73" />